gamla/functional_generic.py

```diff
@@ -89,6 +89,15 @@ def _any_is_async(funcs):
 
 
 async def to_awaitable(value):
+    """Wraps a value in a coroutine.
+    If value is a future, it will await it. Otherwise it will simply return the value.
+    Useful when we have a mix of coroutines and regular functions.
+
+    >>> await to_awaitable(5)
```

Comment:
Consider using `run_sync` to make the example runnable, like here 
https://github.com/hyroai/gamla/blob/master/gamla/functional_async.py#L30